tarodb/sqlite/queries/assets.sql

```diff
@@ -284,7 +282,9 @@ JOIN chain_txns txns
 -- make the entire statement evaluate to true, if none of these extra args are
 -- specified.
 WHERE (
-    assets.amount >= COALESCE(sqlc.narg('min_amt'), assets.amount)
+    assets.amount >= COALESCE(sqlc.narg('min_amt'), assets.amount) AND
+    ((length(hex(sqlc.narg('key_fam_filter'))) == 0 OR 
```

Comment:
I'm curious: Does this work if you do the following instead?
```
key_fam_info_view.tweaked_fam_key = sqlc.narg('key_fam_filter') OR sqlc.narg('key_fam_filter') IS NULL
```

Reply:
Yep! I think you were right about the ordering. I tried to make it short circuit with the `NULL` clause first, but that's what caused it to emit an `interface{}` instead. Tacked that on as another commit.